src/components/ui/button.tsx

```diff
@@ -12,7 +12,7 @@ const buttonVariants = cva('overflow-hidden inline-block text-center', {
     size: {
       default: 'py-3.5 px-5 rounded-2xl',
       icon: 'py-1 px-2 rounded-lg text-xs',
-      theme_icon: 'h-10 w-10',
+      theme_icon: 'p-[0.1875rem] rounded-lg',
```

Comment:
Only one question - if u changed old theme-toggler styles,  do we need in this size variant? As for me, default icon property should be enough(btw we could simply rewrite it's styles to "p-1 rounded-lg"

![check-theme](https://github.com/dev-KPI/messenger-frontend/assets/91133955/a12c7841-6598-483e-9cf9-77a4b5941c5b)

Reply:
Second idea with `p-1 rounded-lg` seems good, I'll fix it